JSONUtilities/Dictionary+JSONKey.swift

```diff
@@ -113,6 +113,31 @@ extension Dictionary where Key: StringProtocol {
     return decodableObjectsArray(jsonArray)
   }
   
+  // MARK: Raw Representable
+  
+  /// Decode an optional RawRepresentable
+  public func jsonKey<ReturnType : RawRepresentable>(key: Key) -> ReturnType? {
+    guard let rawValue = self[key] as? ReturnType.RawValue else {
+      return nil
+    }
+    return ReturnType(rawValue:rawValue)
+  }
+  
+  /// Decode a mandatory RawRepresentable
+  public func jsonKey<ReturnType : RawRepresentable where ReturnType.RawValue:JSONRawType>(key: Key) throws -> ReturnType {
+    
+    guard let rawValue = self[key] as? ReturnType.RawValue else {
```

Comment:
Now that travis is working I noticed that his path is not tested, could you add a unit test case for this? 
https://codecov.io/gh/lucianomarisi/JSONUtilities/src/e0e1ea76bdd60869b48d9c002a5198b7bfc694b8/JSONUtilities/Dictionary%2BJSONKey.swift